statement.Rmd

```diff
@@ -2,28 +2,50 @@
 - [OxCal](https://c14.arch.ox.ac.uk/oxcal.html)
 - ...
 
-The scopes and purposes of these scripting languages are highly different. Some of them are fully capable programming languages which provide methods and libraries for all kind of cross-platform system- and network-interaction. Some are rather specific collections of basic commands to control individual software tools and therefor serve as mere interfaces.
+The scopes and purposes of these scripting languages are highly different. Some of them (like R or Python) are fully capable programming languages which provide methods and libraries for all kind of cross-platform system- and network-interaction. Some (like OxCal) are rather specific collections of basic commands to control individual software tools and therefore serve as mere interfaces. Despite this differences both kind of scripting languages allow to create a reproducible representation of a scientific task.
 
 ### Reproducible research
 
-The use of scripting languages for research allows the user to document every necessary step in a research pipeline. The resulting script can be used to reproduce the data analysis by other researchers, given that also the input datasets are provided. Scripting languages are therefore the ideal environment for reproducible research. R and and Python for example provide specialised frameworks to combine data, code and text (R Markdown, Jupyter Notebooks, ...). Reproducible research is important for archaeology because it allows other researchers to verify published findings, and to easily extend new methods to other datasets. 
+The use of scripting languages for research allows the user to document every necessary step in a research pipeline. The resulting script can be used to reproduce the data analysis by other researchers, given that also the input datasets are provided and -- if necessary -- the required software environment can be emulated. Scripting languages are therefore the ideal for reproducible research. R and and Python for example provide specialised frameworks to combine data, code and text (R Markdown, Jupyter Notebooks, ...). Reproducible research is important for archaeology because it allows other researchers to verify published findings, and to easily extend new methods to other datasets. 
 
 ### Version control
 
 Reproducible research ideally also documents the science production process with all inductive and deductive steps of hypothesis building. A modern and powerful way to store a commented succession of changes in text and scripted data analysis is available with tools like Git or SVN which where initially designed for software development. They provide a simple environment to keep track of even minor changes, go back to earlier working stages and allow for well documented collaborations thanks to forking. The combination of scripting languages and version control is an important basis for a fully comprehensible research process.
 
+### Objectives of the SIG-SSLA
```

Comment:
High expectations, but why not. Might be altered in the course of discussion and by reality later on

Reply:
Ja - let's wait for the feedback.